css/styles.css

```diff
@@ -57,3 +57,56 @@ textarea {
   font-weight: bold;
   letter-spacing: 0.05rem;
 }
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.nav-style {
+  padding-left: 2rem;
+  text-decoration: none;
+  font-size: 1.3rem;
+  background-color: #562b81;
+  border: none;
+}
+
+.align {
+  margin-left: 25rem;
```

Comment:
WOAH this is a huge margin. Why is this necessary? There is almost certainly a better way to accomplish this, likely with `justify-content` or `text-align`

Reply:
fixed.